COVERAGE.md

```diff
@@ -36,17 +36,17 @@ We currently cover the following components:
     -   [] InfoLabel
```

Comment:
Great work! Could you also add the rule to the index.ts file?

Reply:
Done!